src/locales/en.json

```diff
@@ -106,6 +106,7 @@
   "alerts": {
     "clearReset": "This will clear the whole canvas. Are you sure?",
     "couldNotCreateShareableLink": "Couldn't create shareable link.",
+    "couldNotCreateShareableLinkTooBig": "Sorry we couldn't create your shareable link, your canvas is too big",
```

Comment:
```suggestion
    "couldNotCreateShareableLinkTooBig": "Couldn't create shareable link: the scene is too big",
```

Reply:
"canvas" too big isn't really the problem here, it has more to do with payload size. Not that users will appreciate this distinction, but we usually use "scene" anyway.